generic/SpatialConvolutionLocal.c

```diff
@@ -0,0 +1,414 @@
+#ifndef TH_GENERIC_FILE
+#define TH_GENERIC_FILE "generic/SpatialConvolutionLocal.c"
+#else
+
+#ifdef _WIN32
+# include <windows.h>
+#endif
+
+
+
+/* note: due to write issues, this one cannot be parallelized as well as unfolded_copy */
+static void nn_(spatialconvolutionlocal_unfolded_acc)(THTensor *finput, THTensor *input,
```

Comment:
I think it would be better to factor out `unfolded_acc` and `unfolded_copy` in a separate file, as done in the GPU version, to avoid redundant code.